src/App.css

```diff
@@ -5,244 +5,335 @@
+    height: 100vh;
+    justify-content: center;
+    align-items: center;
+    background-color: #000;
+    font-family: Open Sans, sans-serif;
 }
 
+
 .skins-button {
-  color: #000;
-  font-size: 4rem;
-  position: absolute;
-  top: 10%;
-  left: 10%;
+    color: rgb(0, 0, 0);
+    font-size: 4rem;
+    position: absolute;
+    top: 10%;
+    left: 10%;
 }
 
 .gameplay-button {
-  color: #000;
-  font-size: 4rem;
-  position: absolute;
-  bottom: 10%;
-  left: 45%;
+    color: rgb(0, 0, 0);
+    font-size: 4rem;
+    position: relative;
+    top: 60px;
 }
 
 .upgrades-button {
-  color: #000;
-  font-size: 4rem;
-  position: absolute;
-  top: 10%;
-  right: 10%;
+    color: rgb(0, 0, 0);
+    font-size: 4rem;
+    position: absolute;
+    top: 10%;
+    right: 10%;
 }
 
 .userInput {
-  position: absolute;
-  top: 35%;
-  left: 33%;
+    color: rgb(80, 202, 86);
+    position: absolute;
+    top: 45%;
+    width: 100%;
 }
 
 .inputText {
-  font-family: 'Quicksand', sans-serif;
-  font-size: 150%;
+    font-family: 'Quicksand', sans-serif;
+    font-size: 150%;
 }
 
 .inputBox {
-  font-family: 'Quicksand', sans-serif;
-  font-size: 150%;
-  border-radius: 15px;
+    font-family: 'Quicksand', sans-serif;
+    font-size: 150%;
+    border-radius: 15px;
 }
 
 
 .slider {
-  position: relative;
-  height: 100vh;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  /* background-color: #8FF4F4; */
+    position: relative;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    /* background-color: #8FF4F4; */
 }
 
 .image {
-  height: 600px;
-  border-radius: 10px;
+    height: 600px;
+    border-radius: 10px;
 }
 
 .right-arrow {
-  position: absolute;
-  top: 50%;
-  right: 32px;
-  font-size: 3rem;
-  color: #000;
-  z-index: 10;
-  cursor: pointer;
-  user-select: none;
+    position: absolute;
+    top: 50%;
+    right: 32px;
+    font-size: 3rem;
+    color: #000;
+    z-index: 10;
+    cursor: pointer;
+    user-select: none;
 }
 
 .left-arrow {
-  position: absolute;
-  top: 50%;
-  left: 32px;
-  font-size: 3rem;
-  color: #000;
-  z-index: 10;
-  cursor: pointer;
-  user-select: none;
+    position: absolute;
+    top: 50%;
+    left: 32px;
+    font-size: 3rem;
+    color: #000;
+    z-index: 10;
+    cursor: pointer;
+    user-select: none;
 }
 
 .home-button {
-  color: #000;
-  /* position: fixed; */
-  font-size: 3rem;
+    color: #000;
+    /* position: fixed; */
+    font-size: 3rem;
 }
 
+/* gameplay */
+/* .game-screen {
+    background: #8FF4F4;
+} */
 
-.slide {
-  opacity: 0;
-  transition-duration: 1s ease;
-}
-
-.slide.active {
-  opacity: 1;
-  transition-duration: 1s;
-  transform: scale(1.08);
-}
-
-.title {
-  position: absolute;
-  top: 6%;
-  width: 80%;
-  text-align: center;
-  font-size: 2em;
-  font-family: 'Quicksand', sans-serif;
+.gameplay-div {
+    height: 100vh;
+    background-image: url("https://i.imgur.com/ISS5Jow.jpeg");
```

Comment:
Again, I would recommend retrieved a image from a local file, instead of online

Reply:
I want to say I liked the choice to darken the background here, as that helps the player focus on the game. To add to Nam's simplicity advice, I would advise using a linear gradient to darken the original background image rather than linking to a separate, preprocessed image.